packages/gatsby/src/bootstrap/load-plugins/load.js

```diff
@@ -159,5 +158,29 @@ module.exports = async (config = {}) => {
     },
   })
 
+  // Add the auto page creator plugin
+  const pluginPageCreator = {
+    version: require(
+      path.join(
+        path.dirname(
+          require.resolve(`gatsby-plugin-page-creator`)), `package.json`
+        )
+      ).version,
```

Comment:
Not sure what better resolution patterns exist in this project.

Reply:
I think instead of manually creating that you could reuse `processPlugin` function - something like that:
```js
plugins.push(processPlugin({
  resolve: `gatsby-plugin-page-creator`,
  options: {
    path: path.join(program.directory, `src/pages`)
  }
}))
```
(check comment below for program.directory)